v4l2.go

```diff
@@ -311,6 +334,54 @@ func getFrameSize(fd uintptr, index uint32, code uint32) (frameSize FrameSize, e
 	return
 }
 
+func getFrameInterval(fd uintptr, index uint32, code uint32, width uint32, height uint32) (FrameRate, error) {
+	frmivalenum := &v4l2_frmivalenum{}
```

Comment:
better to initialize in struct and use camel casing.

Reply:
Switched to camelcase and initialize in struct.